### DISCUSSION TITLE
Possible to get Mesa Zink (OpenGL to Vulkan) working with WINE (Wineskin) ?

### DISCUSSION THREAD

**Discussion by sofakng** [The-Wineskin-Project/WineskinServer#464] [General]
Is it possible to get Mesa Zink (OpenGL to Vulkan) working on WINE (Wineskin) ?  My goal would be OpenGL -> Zink (Vulkan) -> MoltenVK...

I'm trying to follow these instructions:  https://www.reddit.com/r/macgaming/comments/126xvrq/running_modern_opengl_windows_games_on_crossover/

...but it doesn't seem to work correctly.

Has anybody else tried and had any success?

**Comment by Gcenx** [comment]
You can’t MoltenVK doesn’t provided enough Vulkan extensions to make proper use of Zink.

Some CodeWeavers staff had looked into the possibility of using Zink over macOS OpenGL, Zink was much slower than OpenGL over Metal on Apple Silicon and worse overall.

If we’re lucky Zeb will be done with there dx9 over Vulkan work by CX24.
